backend/courses/models.py

```diff
@@ -366,6 +366,36 @@ class Topic(models.Model):
         ),
     )
 
+    historical_probabilities_spring = models.FloatField(
+        default=0,
+        help_text=dedent(
```

Comment:
wait why do we need 3 things here. We should just have a single field for the expected semester or unknown (based on some criteria) unless you think there's a strong reason to split it into 3 fields.

Reply:
I think the idea behind this was for the case that even if the expected semester is to capture seasonal classes:
(Ex) If expected semester is Spring 2025, we would have to account for the for the case it is not offered in Fall 2025?